api/app/signals/settings/testing.py

```diff
@@ -10,17 +19,20 @@
 ]
 CORS_ALLOW_ALL_ORIGINS = False
 
-DATABASES['default']['HOST'] = 'database'  # noqa: F405
-DATABASES['default']['PORT'] = '5432'  # noqa: F405
+DATABASE_HOST = os.getenv('DATABASE_HOST', 'database')  # noqa: F405
```

Comment:
No need for the `# noqa: F405` here. F405 `name may be undefined, or defined from star imports` was used because the `DATABASES['default']['HOST']` was imported and updated. This is no longer the case so the `# noqa: F405` can be removed.